### PR TITLE
Add docstring for ApplicantProgramBlocksController

### PR DESCRIPTION
**Pull request by hmhsu**
### Description
Add docstring and minor refactor to make ApplicantProgramBlocksController more readable

### CODE REVIEW

File: universal-application-tool-0.0.1/app/controllers/applicant/ApplicantProgramBlocksController.java [hunk 1]
```diff
@@ -79,12 +78,31 @@ public ApplicantProgramBlocksController(
...
     return editOrReview(request, applicantId, programId, blockId, false);
   }
 
+  /**
+   * This method renders all questions in the block of the program and presents to the applicant.
+   *
+   * <p>The difference between `edit` and `review` is the next block the applicant will see after
+   * submitting the answers.
+   *
+   * <p>`review` takes the applicant to the first imcomplete block. If there are no more blocks,
```
Comment (carolinedanzi) [inline universal-application-tool-0.0.1/app/controllers/applicant/ApplicantProgramBlocksController.java:103]:
```suggestion
   * <p>`review` takes the applicant to the first incomplete block. If there are no more blocks,
```

File: universal-application-tool-0.0.1/app/controllers/applicant/ApplicantProgramBlocksController.java [hunk 2]
```diff
@@ -301,7 +312,7 @@ public ApplicantProgramBlocksController(
         .exceptionally(ex -> handleUpdateExceptions(ex));
   }
 
-  private CompletionStage<Result> update(
+  private CompletionStage<Result> renderErrorOrRedirectToNextBlock(
```
Comment (carolinedanzi) [inline universal-application-tool-0.0.1/app/controllers/applicant/ApplicantProgramBlocksController.java:315]: praise: love the name change here!

**Review by carolinedanzi — APPROVED** [review]
Thank you Hung-Ming, this is much nicer!

**Review by kendrickt — APPROVED** [review]

File: universal-application-tool-0.0.1/app/controllers/applicant/ApplicantProgramBlocksController.java (hunk 2 above)
Comment (kendrickt) [reply universal-application-tool-0.0.1/app/controllers/applicant/ApplicantProgramBlocksController.java:315]: +1

**Pull request merged**